apis/core/bootstrap/shapes/table.ts

```diff
@@ -36,10 +36,11 @@ ${shape('table/create')} {
     ] ;
     ${sh.property} [
       ${sh.name} "Identifier template" ;
-      ${sh.description} "Used to build a unique identifier for each row of this table. Leave empty to get an auto-generated identifier." ;
+      ${sh.description} "Used to build a unique identifier for each row of this table. Not supported yet: Leave empty to get an auto-generated identifier." ;
       ${sh.path} ${cc.identifierTemplate} ;
       ${sh.minCount} 1 ;
       ${sh.maxCount} 1 ;
+      ${sh.minLength} 1 ;
```

Comment:
Tell you what, a `sh:pattern "[\w\d]+"` would be event better, to prevent whitespace-only

Reply:
Pattern is happy when the RegExp finds a matching pattern in the string or does the entire string match

Reply:
Either `[\w\d]` is sufficient or it would not work since there will be `-` and  `{}`in the entry

Reply:
Aha, good point. For entire value you'd need `^` and `$`

Reply:
Until we know exactly what is needed i'll go with `sh:pattern "[\w\d]"`

Reply:
>Either [\w\d] is sufficient or it would not work since there will be - and {}in the entry

Good point. Might as well run with `.+` to avoid going into template syntax specific.